### DISCUSSION TITLE
[WebProfilerBundle] Randomly Exceptions

### DISCUSSION THREAD

**Discussion by benjaminfunk** [symfony/symfony#50816] [Q&A] [answered]
### Symfony version(s) affected

6.3.0

### Description

After i upgrade to v6.3 i get randomly Exceptions.

I has something to do with this feature: https://github.com/symfony/symfony/pull/47352

The Exception always comes with another Exception (look screenshots)

Maybe it has something to do with Windows

### How to reproduce

Simply load the page more often so that the coincidence occurs

### Additional Context

<img width="805" alt="Screenshot 2023-06-25 131128" src="https://github.com/symfony/symfony/assets/856290/507f519d-229f-4f39-9e17-2387d3b49460">

<img width="826" alt="Screenshot 2023-06-25 131205" src="https://github.com/symfony/symfony/assets/856290/86612af8-bcff-49a6-b860-6b2ff24550d8">

**Comment by benjaminfunk** [comment]
Was a bad index.csv. Presumably the cache wasn't cleared after the upgrade, so the index had fewer values than needed

**Reply by derrabus** [reply]
You should always clear your cache after changing dependencies.

**Comment by benjaminfunk** [comment]
Clear cache helps. But it seems that older entries in the index.csv cause problems. As soon as I don't work on the app for a few days, there are random problems again.

**Reply by derrabus** [reply]
I've converted your issue to a discussion. Without a reproducer, your bug report is not actionable, so let's file it as a support request for now.

**Comment by benjaminfunk** [comment]
<img width="989" alt="Screenshot 2023-06-29 104924" src="https://github.com/symfony/symfony/assets/856290/32896a50-1e38-4e2c-afcb-7203a674bd14">

The problem seems to happen as soon as the entries are created. Somehow I have entries in the index.csv that I can't explain.

**Reply by derrabus** [reply]
That broken line looks like a truncated version of the line right below. 🤔

**Reply by benjaminfunk** [reply]
<img width="1046" alt="Screenshot 2023-06-29 112555" src="https://github.com/symfony/symfony/assets/856290/0e28eeaa-209a-41a9-908b-e312f63ce6f6">

I think the file itself doesn't seem to be the problem as it works fine in other cases.

**Reply by benjaminfunk** [reply]
<img width="920" alt="Screenshot 2023-06-29 113243" src="https://github.com/symfony/symfony/assets/856290/ad8d6b25-bf9e-43c4-83fe-73edb4652e55">

Found another case where the broken line is like a truncated version of the lines above/under.

**Reply by derrabus** [reply]
Can you somewhat track down (e.g with a step debugger) in what situations that broken line is written?

**Reply by benjaminfunk** [reply]
I'll try to find out something. I currently have 4000 entries in my index.csv and only 3 incorrect lines. Could also take a while :P

**Reply by derrabus** [reply]
Sure, take your time.

**Comment by benjaminfunk** [comment, marked as answer]
I have now collected over 4000 new records without having an error. If I remember correctly, I have updated to version 6.3.1 in the meantime. Maybe that fixed the problem. If the error occurs again, I will open this discussion again.

**Reply by derrabus** [reply]
Thanks for the heads-up. Yes, please do that.
